.github/workflows/run-tests-windows.yml

```diff
@@ -92,6 +92,13 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      # We need to set proper Pagefile limits in the advance. 
+      - name: Configure Pagefile
+        uses: al-cheb/configure-pagefile-action@v1.2
+        with:
+          minimum-size: 4GB
+          maximum-size: 16GB
```

Comment:
Why is this necessary? Does it mean that people using SN on their systems must also do something like that?